src/backtrace/dbghelp64.rs

```diff
@@ -127,5 +130,9 @@ pub unsafe fn trace(cb: &mut dyn FnMut(&super::Frame) -> bool) {
             &mut establisher_frame,
             ptr::null_mut(),
         );
+
+        if context.ip() == 0 || (context.ip() == previous_ip && context.sp() == previous_sp) {
```

Comment:
At various points in this loop, `context.ip()` is called. Can you `let`-bind it at the very first chance in this loop, and then `let`-bind it again immediately after the value changes, so that it's obvious at which point that happens? Same with the stack pointer.

Reply:
Sure